documentation/modules/configuring/con-considerations-for-data-storage.adoc

```diff
@@ -28,6 +29,19 @@ Consider the underlying architecture and requirements of your deployment when ch
 
 For more information, refer to {ApacheKafkaFileSystem} in the Kafka documentation.
 
+== Tiered storage
+
+Kafka’s tiered storage feature is supported in Strimzi as an optional capability. 
+
+With tiered storage enabled:
+
+* Primary broker storage,such as persistent volumes or JBOD, handles recent data
+* Remote tiered storage, such as object storage, is used for historical data
+
+AWS S3 and NFS-backed storage are supported for the remote tier.
+Other S3-compatible or cloud-based object storage systems might also work, but they have not been tested.
+If you use an untested storage backend, validate its performance and compatibility in your environment before using it in production.
```

Comment:
This is not true. Strimzi does not provide any support for S3 or NFS. Strimzi allows you to bring your own plugin with whatever that plugin supports. So this paragraph should be removed.

Reply:
Yes, we don't ship any tiered storage plugin libs in Strimzi.

Reply:
Thanks. I've removed and left some general guidance